web/.eslintrc

```diff
@@ -1,51 +1,92 @@
 {
+  "plugins": [
+    "prettier",
+    "@typescript-eslint",
+    "jest"
+  ],
   "extends": [
-    "eslint:recommended",
-    "plugin:react/recommended"
+    "airbnb-typescript",
+    "prettier",
   ],
-  "parser": "@babel/eslint-parser",
+  "parser": "@typescript-eslint/parser",
   "parserOptions": {
-    "ecmaVersion": 2017,
-    "ecmaFeatures": {
-      "jsx": true,
-      "experimentalDecorators": true,
-      "legacyDecorators": true,
-      "decoratorsBeforeExport": true
-    },
-    "sourceType": "module"
+    "project": "./tsconfig.json"
   },
-  "env": {
-    "browser": true,
-    "node": true,
-    "es6": true
+  "settings": {
+    "import/resolver": {
+      "typescript": {
+        "alwaysTryTypes": true
+      }
+    }
   },
-  "plugins": [
-    "react"
-  ],
   "rules": {
-    "quotes": ["error", "double", { "allowTemplateLiterals": true }],
-    "indent": "off",
-    "no-console": [0],
-    "no-debugger": [0],
+    "object-curly-spacing": [
+      "warn",
+      "always"
+    ],
     "no-unused-vars": "off",
```

Comment:
we don't use the eslint `no-unused-vars` because it doesn't work with typescript. the typescript-eslint plugin will handle this for us now